portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that deletes all comments from the Datastore. */
+@WebServlet("/delete-comments")
+public class DeleteCommentsServlet extends HttpServlet {
+  private static DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  /**
+   * {@inheritDoc}
+   * 
+   * <p>This Method handles POST requests corresponding to deleting all Comment kind Entities from the
+   * Google Cloud Datastore.
+   *
+   * <p>The POST request also results in a re-direct back to the original server-dev page.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Key> entityKeys = new ArrayList<>();
+    for (Entity commentEntity : results.asIterable()) {
+      entityKeys.add(commentEntity.getKey());
+    }
```

Comment:
this is totally fine. can be written as 
```suggestion
    results.forEach(entityKeys::add);
```

Reply:
Ohh this is nice. It amost reminds me of python list comprehension.

Reply:
I actually just found that in order for it to work, I have to do this

`results.asIterable().forEach(commentEntity -> entityKeys.add(commentEntity.getKey()));`

Do you prefer this over the original still?

Reply:
Chatted offline. I am ok with the existing implementation, will dig more when I have more time here.

Reply:
Yes per our chat conversion. Resolved.